platformio.ini

```diff
@@ -21,35 +21,44 @@
 env_default = emonesp
 data_dir = src/data
 
+[common]
+version = -DBUILD_TAG=2.0.0
+#MQTT: https://github.com/knolleary/pubsubclient
+lib_deps = PubSubClient@2.6
```

Comment:
Awesome! 👍  Love the new PlatformIO library manager. It's a step change in firmware build future replicability to be able to directly reference library versions. 

As I side note I have been in discussion with Ivan from platformIO regarding the pros and cons of using library ID vs Names vs direct GitHub reference: https://community.platformio.org/t/3-1-0-library-manager-installing-from-git/806